draft-ietf-core-resource-directory.md

```diff
@@ -529,7 +529,7 @@ Req: GET coap://[ff02::1]/.well-known/core?rt=core.rd*
 Res: 2.05 Content
 </rd>;rt="core.rd";ct=40,
 </rd-lookup/ep>;rt="core.rd-lookup-ep";ct=40,
-</rd-lookup/res>;rt="core.rd-lookup-res";ct="40",
```

Comment:
While this is indeed unnecessary, it does show that the quotes are legitimate (see Figure 12 in Section 7.2.1 of RFC 7252:

```
      ct-value =  cardinal
               /  DQUOTE cardinal *( 1*SP cardinal ) DQUOTE
```
)